test/test_methods.py

```diff
@@ -149,6 +149,29 @@ def test_load_coordinates(self):
                 for coords, expected in zip(coordinates, objects):
                     self.assertEqual(coords, expected)
 
+    def test_load_coordinates_scientific_notation(self):
+
+        # For some datasets that generate coords so close to zero that they end up in scientific notation
+
+        data = '7.9720694373e-05 44.6352243008'
+        with methods.tempinput(data) as path:
+            coordinates = methods.load_coordinates(path)
+            coords_list = list(coordinates)
+            self.assertEqual(len(coords_list), 1)
+            self.assertEqual(coords_list[0], (7.9720694373e-05, 44.6352243008, None, None))
+
+    def test_load_coordinates_scientific_notation_with_propper_motion(self):
+
+        # For some datasets that generate coords so close to zero that they end up in scientific notation
```

Comment:
This is the same comment as above, so I don't think we need it.

Reply:
removed